docs/gettingstarted/buildspecs_interface.rst

```diff
@@ -276,6 +279,9 @@ display the following information
 
     .. command-output:: buildtest buildspec maintainers --breakdown
 
+If you want to see the table output of ``buildtest buildspec maintainers --breakdown`` with a specific color option you can use ``buildtest --color=<COLOR> buildspec maintainers --breakdown``
+command.
```

Comment:
I am thinking we can drop these sentences, i think having to document a sentence for every `buildtest --color <COLOR> <sub-command>` can be a bit redundant so id be fine with removing this.

Reply:
@shahzebsiddiqui Yes, makes sense. I have removed those sentences.